content/runner/vm/installation.md

```diff
@@ -10,12 +10,11 @@ description: |
 ---
 
 <div class="alert">
-The AWS runner is in the Release Candidate phase.
+The VM runner is in the Release Candidate phase.
 </div>
 
-This article explains how to install the runner on Linux. The AWS runner is packaged as a minimal Docker image distributed on [DockerHub](https://hub.docker.com/r/drone/drone-runner-aws).
-**By default it will use Amazon EC2, with a max pool size of 2 instances running Ubuntu 18.04, in a pool called `test_pool`.**
-Currently the runner is only supports amazon, but in the future it will support other cloud providers and virtual machines.
+This article explains how to install the runner on Linux. The VM runner is packaged as a minimal Docker image distributed on [DockerHub](https://hub.docker.com/r/drone/drone-runner-aws).
+
```

Comment:
we are missing information on what the runner does by default. this can be moved into the aws section

Reply:
done